src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -13,13 +17,39 @@
     public static final String COMMAND_WORD = "list";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Displays all persons in the address book as a list with index numbers.\n"
+            + ": Displays all persons in the address book as a list with index numbers in alphabetical order.\n"
             + "Example: " + COMMAND_WORD;
 
-
+    /**
+     * Lists the entries in the addressbook in alphabetical order according to their name.
+     */
     @Override
     public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        List<ReadOnlyPerson> listOfSortedPersons = orderListOfEntries(allPersons);
+        return new CommandResult(getMessageForPersonListShownSummary(listOfSortedPersons), listOfSortedPersons);
+    }
+
+    /**
+     * Receives an unordered list of entries in an addressbook and orders them in alphabetical order
+     */
+
+    public List<ReadOnlyPerson> orderListOfEntries(List<ReadOnlyPerson> unorderedListOfPersons)
+    {
+        List<ReadOnlyPerson> listOfSortedPersons = new ArrayList<>();
+        for (int i = 0; i < unorderedListOfPersons.size(); i++) { // Copy List from unsorted array to array that will be sorted
+            listOfSortedPersons.add(unorderedListOfPersons.get(i));
+        }
+
+        for (int i = 0; i < listOfSortedPersons.size()-1; i++)
+            for (int j = 0; j < listOfSortedPersons.size()-i-1; j++)
```

Comment:
Violates our coding standard as there is no curly brackets. Moreover, I believe there should be empty space between symbols. Check the coding standard.